Directory.Packages.props

```diff
@@ -8,12 +8,12 @@
     <PackageVersion Include="Acornima.Extras" Version="1.1.1" />
     <PackageVersion Include="BenchmarkDotNet" Version="0.14.0" />
     <PackageVersion Include="BenchmarkDotNet.TestAdapter" Version="0.13.12" />
-    <PackageVersion Include="FluentAssertions" Version="7.0.0" />
+    <PackageVersion Include="FluentAssertions" Version="8.0.0" />
```

Comment:
🤦

Reply:
Free for open source at least... Otherwise I would pin to 7.x